stacks-signer/src/chainstate.rs

```diff
@@ -113,6 +113,18 @@ impl SortitionsView {
         block: &NakamotoBlock,
         block_pk: &StacksPublicKey,
     ) -> Result<bool, ClientError> {
+        let bitvec_all_1s = block.header.pox_treatment.iter().all(|entry| entry);
```

Comment:
So the signer policy for now is to just have all 1's?

Reply:
Yep